### DISCUSSION TITLE
Обновление на сайте и документации

### DISCUSSION THREAD

**Discussion by LostAlly** [totumonline/totum-ru-issues-and-discussions#556] [Неопределенное]
Я плохо искал или такой раздел отсутствует?
Было бы здорово видеть какие разделы добавлены на сайт и в документацию, какие ошибки исправленны в документации )
Одним словом хотелось бы ченждж лог какой то по сайту.
Спасибо.

**Comment by ttm-alexeypolunin** [comment]
Его не будет по причине отсутствия ресурсов... Посмотрим летом, что и в каком количестве будут платить за лицензии. Будет хороший вариант, если мы наберем на работу хотя бы половины программиста.
